frontend/src/Store/Selectors/createHealthCheckSelector.js

```diff
@@ -11,8 +12,8 @@ function createHealthCheckSelector() {
         items.push({
           source: 'UI',
           type: 'warning',
-          message: 'Could not connect to SignalR, UI won\'t update',
-          wikiUrl: 'https://github.com/Radarr/Radarr/wiki/Health-Checks#could-not-connect-to-signalr'
+          message: translate('CouldNotConnectSignalR'),
```

Comment:
Might need to wrap this in `{ }`

Reply:
might or do lol? 
neither lint nor build complained?

Reply:
I think this is good looking at it closer